sdk/core/azure-core-amqp/src/test/java/com/azure/core/amqp/implementation/ReactorConnectionTest.java

```diff
@@ -513,7 +521,7 @@ void createCBSNodeTimeoutException() throws IOException {
 
                 final AmqpException amqpException = (AmqpException) error;
                 assertTrue(amqpException.isTransient());
-                assertNull(amqpException.getErrorCondition());
+                assertEquals(TIMEOUT_ERROR, amqpException.getErrorCondition());
```

Comment:
Note: Now that timeout for connection to active uses TIMEOUT_ERROR condition, assert the same.